graylog2-web-interface/src/components/indices/IndexSummary.jsx

```diff
@@ -61,13 +72,10 @@ const IndexSummary = React.createClass({
     return (
       <span>
         <h2>
-          {this._deflectorBadge()}{' '}
-
-          {index.name}{' '}
-
-          {this._reopenedBadged(index)}{' '}
+          {this.props.name}{' '}
 
           <small>
+            {this._formatLabels(index)}{' '}
             {this._formatIndexRange(index)}{' '}
 
             <IndexSizeSummary index={index} />
```

Comment:
`IndexSizeSummary` is not updated to the new data structure coming from the API, and it's never rendered.